lib/clearance/session.rb

```diff
@@ -14,15 +14,9 @@ def initialize(env)
     # Called by {RackSession} to add the Clearance session cookie to a response.
     #
     # @return [void]
-    def add_cookie_to_headers(headers)
+    def add_cookie_to_headers(_headers)
```

Comment:
just realized that this is being called from `lib/clearance/rack_session.rb` so we can actually remove the headers param which is no longer needed.